src/AlreadyConstructedMarker.js

```diff
@@ -4,6 +4,7 @@
  * element construction stacks.
  *
  * https://html.spec.whatwg.org/#concept-already-constructed-marker
+ * @extends AlreadyConstructedMarkerType
  */
 class AlreadyConstructedMarker {}
```

Comment:
Not really important but I just realized we don't actually need a class + instance of it here:
```js
export default /** @type {!AlreadyConstructedMarkerType} */ ({});
```